programs/nodeos/CMakeLists.txt

```diff
@@ -1,3 +1,11 @@
+include(additionalPlugin)
+
+foreach(ADDITIONAL_PLUGIN_SOURCE_DIR ${EOSIO_ADDITIONAL_PLUGINS})
+    string(UUID ADDITIONAL_PLUGIN_SOURCE_DIR_MD5 NAMESPACE "00000000-0000-0000-0000-000000000000" NAME ${ADDITIONAL_PLUGIN_SOURCE_DIR} TYPE MD5)
```

Comment:
I like where you went with this, should normalize the input first via `get_filename_component( ... ABSOLUTE ... )` ?

 I assume that ".././my_plugin", "../my_plugin", "../my_plugin/../my_plugin", "..//my_plugin" etc are all going to have different UUIDs if we don't normalize them.

Reply:
I don't think it's necessary: cmake will fail because you got targets with same name.